### ISSUE TITLE
Natura interaction: Exoflame dupe

### ISSUE THREAD

**Issue by yrsegal** [Vazkii/Botania#674]
This is a glitch with Natura's Nether Furnace.

If you have a Nether Furnace near an Exoflame, and you put something in the Nether Furnace to smelt, it will turn into a normal furnace, toss out all the items it was smelting, and have an extra one of them in the furnace. If you put the two stacks together, they'll go back to what they were before: but if you leave the one item in the furnace smelting, you end up with a net positive of 1.

TL;DR
Exoflame turns nether furnace into normal furnace duping things in the process.

Using Direwolf20, 1.0.1
